backend/database.js

```diff
@@ -84,6 +84,47 @@ export default (sbp('sbp/selectors/register', {
       }
     })
   },
+  'backend/db/streamEntriesBetween': async function (startHash: string, endHash: string, offset: number): Promise<*> {
+    offset = isNaN(parseInt(offset)) ? 0 : Math.max(0, parseInt(offset))
```

Comment:
`offset` is defined as type `number`, so there is no reason to use `parseInt` here. See other comment in `routes.js`.

Reply:
Okay.